conftest.py

```diff
@@ -1,13 +1,86 @@
+import datetime
 import logging
 import os
 import platform
+import re
+import sys
 from typing import Callable, List, Tuple
 
 import pytest
 from selenium import webdriver
 from selenium.common.exceptions import WebDriverException
+from selenium.webdriver import Firefox
 from selenium.webdriver.firefox.options import Options
 
+# pipenv run pytest --fx-executable ./firefox/firefox -n 4 .
+
+
+def screenshot_content(driver: Firefox, opt_ci: bool, test_name: str) -> None:
```

Comment:
So...how does this differ from the `screenshot` fixture? (Especially given that we know that screenshotting the chrome context doesn't necessarily work)

Reply:
so for the screenshot fixture, if I request that fixture I end up get a recursive dependency since I need the driver, perhaps there's a better way to do it that you are referring to? I was kind of on edge about this code cuz I basically rewrote your fixture